README.md

```diff
@@ -1 +1,33 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello Microverse Project
+> This is a simple Hello World Project
+
+## Built With
+- HTML
+## Getting Started
+
+**Download the .zip file and extract it.**
+**Open the index.html file with your browser.**
+
+## Authors
+
+
+👤 **Sinan Sevgi**
+
+- GitHub: [@sinansevgi](https://github.com/sinansevgi)
+- LinkedIn: [@sinansevgi](https://www.linkedin.com/in/sinan-s-52559437/)
+- Twitter: [@nativeofcyber1](https://twitter.com/nativeofcyber1)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
```

Comment:
- [ ] **Your issue page is broken. Please fix it**

Having broken links can reflects imperfection and inconsistency. It will result in a bad impression to anyone reading your README. Please fix it.